app/services/finance/ecf/statement_calculator.rb

```diff
@@ -131,8 +131,12 @@ def total_for_uplift
         [available, delta_uplift_amount].min
       end
 
+      def uplift_clawback_deductions
```

Comment:
suggestion: maybe we can use this method also here: https://github.com/DFE-Digital/early-careers-framework/blob/main/app/views/finance/ecf/statements/show.html.erb#L200

Reply:
Hi @cwrw spot on.. tks 👍 😄